src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -143,12 +147,23 @@ public function __construct(array $options = [])
 
         $options['error_handler'] ??= SymfonyErrorHandler::class;
 
+        $workerLoopMax = $options['worker_loop_max'] ?? $_SERVER['FRANKENPHP_LOOP_MAX'] ?? $_ENV['FRANKENPHP_LOOP_MAX'] ?? null;
```

Comment:
Shouldn't the env var has the same name as the option?

Reply:
It'd say we can keep FRANKENPHP_LOOP_MAP to help transitioning.
We also have the `$_SERVER['APP_RUNTIME_OPTIONS']` vars which allows configuring any options.

Reply:
Wouldn't that require a modification in FrankenPHP itself?

Reply:
We could make FrankenPHP compatible with both `FRANKENPHP_LOOP_MAX` and (something like) `WORKER_LOOP_MAX`, then deprecated the first one day when we're ready

Reply:
No need to modify FrankenPHP, this option is entirely handled by the runtime.